stdlib/public/core/OptionSet.swift

```diff
@@ -278,7 +278,7 @@ extension OptionSet where Element == Self {
   ///     print(expressOption == .express)
   ///     // Prints "false"
   ///     print(expressOption == .secondDay)
-  ///     // Prints "true"
+  ///     // Prints "false"
```

Comment:
This change isn't right -- the documentation actually described the expected behavior here, even though the code got it wrong in previous versions. (The return value is prescribed by `SetAlgebra`). 

The stdlib used to return the wrong value, but this was fixed on master -- see https://github.com/apple/swift/pull/28378.

Reply:
Thanks. Removed this change/commit.